### DISCUSSION TITLE
Release version numbering

### DISCUSSION THREAD

**Discussion by alexmyczko** [wkentaro/labelme#1263] [Ideas / Feature requests]
Is there a reason for 5.2.0.post4? Why can it just not be 5.2.1 or 5.2.0.1?

**Comment by wkentaro** [comment]
@alexmyczko I originally thought *"I didn't change any code in the Python package"* to fix the release Github Action  so shouldn't increment the version. But technically *"I changed the repo"* and `.github/release.yml`, so maybe it could just be 5.2.1 as you are wondering. 

So I guess the issue was for Linux build and version format? If so, I don't mind stop using `.postX`.

**Reply by wkentaro** [reply]
There was an issue in release build, so `v5.2.0` didn't have executables in the assets.

<img width="939" alt="image" src="https://user-images.githubusercontent.com/4310419/232458163-97ab7c29-50aa-405d-b19c-95a3ce2cc6b8.png">
